src/warc2zim/content_rewriting/html.py

```diff
@@ -215,6 +228,12 @@ def process_attr(
             return (attr_name, self.css_rewriter.rewrite_inline(attr_value))
         if attr_name.startswith("on") and not attr_name.startswith("on-"):
             return (attr_name, self.js_rewriter.rewrite(attr_value))
+        if attr_name == "content" and rewrite_content_attr:
```

Comment:
I find this very fragile and error prone: having the condition split at two different parts of the code.

Reply:
Just right. I found it pretty nasty to write as well. For now I'll just add a comment, but I've opened https://github.com/openzim/warc2zim/issues/305 to refactor this part of the codebase.